google/cloud/datastore_v1/gapic/datastore_client.py

```diff
@@ -464,9 +461,9 @@ def begin_transaction(
     def commit(
         self,
         project_id,
-        mode,
-        mutations,
+        mode=None,
```

Comment:
param reordering poses a breaking change. Likely don't want to merge this pr

Reply:
Reverted the library changes in `google/` to just take template updates.